cardano-cli/src/Cardano/CLI/EraBased/Run/CreateTestnetData.hs

```diff
@@ -500,7 +502,7 @@ updateCreateStakedOutputTemplate
                                 ]
           , sgStaking =
             ShelleyGenesisStaking
-              { sgsPools = ListMap pools
+              { sgsPools = ListMap . Set.toList $ pools
```

Comment:
So we use ListMap for performance reasons. You'll need to ask @newhoggy if this is OK.

Reply:
This could be a problem.  Please avoid conversion from `List` to `Set` and back again, which I think is what happens.  Please also ensure that the list can be evaluated lazily.

Reply:
@newhoggy> I removed the conversion and will benchmark soon :+1: